configs/repo-config.csv

```diff
@@ -1,2 +1,2 @@
-Repository's Location,Branch,File formats,Ignore Glob List,Ignore standalone config,Ignore Commits List,Ignore Authors List
-https://github.com/reposense/RepoSense.git,master,,,,,
+Repository's Location,Branch,File formats,Ignore Glob List,Ignore standalone config,Ignore Commits List,Ignore Authors List,Shallow Cloning,Find Previous Authors
```

Comment:
It looks like a number of columns are still missing, do refer to the documentation on the full list of columns that should be present:
1. File Size Limit
2. Ignore File Size Limit
3. Skip Ignored File Analysis

If possible, do also follow the order that is presented in the user guide so it is easier for our users to follow.

Reply:
Ah, my bad, I was following the config file in the reposense repo. I've updated and ordered them according the the docs. Let me know if you need any more changes.

Reply:
@RyanCheungJF The newline needs to be added as well, currently the `repo-config.csv` file is missing a final newline.